internal/utils/helpers.go

```diff
@@ -57,3 +58,8 @@ func hasPrefix(buf []byte, prefix []byte) bool {
 func CreateWorkDir() (string, error) {
 	return CreateTmpDir(AppFS, afero.GetTempDir(AppFS, ""), "ec-work-")
 }
+
+// ImageRefTransport is used to inject the type of transport to use with the
+// remote.WithTransport function. By default, remote.DefaultTransport is
+// equivalent to http.DefaultTransport, with a reduced timeout and keep-alive
+var ImageRefTransport = remote.WithTransport(remote.DefaultTransport)
```

Comment:
I think it's a bit dangerous to use a public global variable like this. Can we make it private within the `application_snapshot_image` module?

Reply:
ack